src/pip/_internal/cache.py

```diff
@@ -179,24 +200,29 @@ class WheelCache(Cache):
     """
 
     def __init__(self, cache_dir, format_control):
+        # type: (str, FormatControl) -> None
         super(WheelCache, self).__init__(
             cache_dir, format_control, {'binary'}
         )
         self._wheel_cache = SimpleWheelCache(cache_dir, format_control)
         self._ephem_cache = EphemWheelCache(format_control)
 
     def get_path_for_link(self, link):
+        # type: (Link) -> str
         return self._wheel_cache.get_path_for_link(link)
 
     def get_ephem_path_for_link(self, link):
+        # type: (Link) -> str
         return self._ephem_cache.get_path_for_link(link)
 
     def get(self, link, package_name):
+        # type: (Link, Optional[str]) -> Link
         retval = self._wheel_cache.get(link, package_name)
         if retval is link:
             retval = self._ephem_cache.get(link, package_name)
         return retval
 
     def cleanup(self):
+        # type: (...) -> None
```

Comment:
```suggestion
        # type: () -> None
```

The `...` isn't needed here.